### PR TITLE
solana: clean up Makefile and Anchor.toml

### PR DESCRIPTION
**Pull request by a5-pickle**
* Remove build dependencies
* Remove Anchor.toml.base
* Change scaffolding pubkeys
* Clean up Makefile

### CODE REVIEW

File: solana/Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .DEFAULT_GOAL = build
 .PHONY: build test unit-test integration-test clean
 
-ifndef (NETWORK)
-  $(info No NETWORK specified, defaulting to local cluster (devnet))
-  NETWORK = devnet
+ifndef NETWORK
+	$(info No NETWORK specified, defaulting to local cluster (devnet))
+	NETWORK = devnet
```
Comment (barnjamin) [inline solana/Makefile:6]: I think this is breaking makefile parsing

replacing tabs with spaces seems to work
```
ben@Bens-Mac-mini solana % make unit-test
Makefile:20: *** recipe commences before first target.  Stop.
```

Comment (barnjamin) [inline solana/Makefile:6]:
```suggestion
  $(info No NETWORK specified, defaulting to local cluster (devnet))
  NETWORK = devnet
```

**Comment by barnjamin** [comment]
unit tests go fine, now getting: 

```
test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

anchor test --arch sbf
error: command failed: 'cargo': Bad CPU type in executable (os error 86)
make: *** [Makefile:30: integration-test] Error 1
```

**Comment by a5-pickle** [comment]
> unit tests go fine, now getting:
> 
> ```
> test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
> 
> anchor test --arch sbf
> error: command failed: 'cargo': Bad CPU type in executable (os error 86)
> make: *** [Makefile:30: integration-test] Error 1
> ```
> 
> from: https://solana.stackexchange.com/questions/3435/error-command-failed-cargo-bad-cpu-type-in-executable-os-error-86
> 
> ```
> softwareupdate --install-rosetta
> ```
> 
> its now building but the test script fails
> 
> ```
> Running test suite: "/Users/ben/wormhole-scaffolding/solana/Anchor.toml"
> 
> Unable to get latest blockhash. Test validator does not look started. Check .anchor/test-ledger/test-ledger-log.txt for errors. Consider increasing [test.startup_wait] in Anchor.toml.
> make: *** [Makefile:30: integration-test] Error 1
> ```
> 
> This is apparently caused because of an issue with gnu-tar, using the `brew` installed one is the solution: https://solana.stackexchange.com/questions/4499/cant-start-solana-test-validator-on-macos-13-0-1/4761#4761
> ## And last issue seems to be that I need to run `npm install` before trying `make test`
> 
> For all users:
> 
>     * run `npm install` prior to `make test`
> 
> 
> For Mac users:
> 
>     * Brew install `make` > 4.x (and ensure its being used as the default `make` command by adding it to $PATH)
> 
>     * Brew install  `gnu-tar` (for solana-local-validator)

I needed to add `node_modules` as a dependency for the `integration-test` command. Will push a fix soon.

**Review by barnjamin — APPROVED** [review]

**Review by hydrogenbond007 — COMMENTED** [review]
Tests and build working on all macs

**Pull request merged**